tests/e2e/runner/__tests__/minter-suite-v2/ram-lib.test.ts

```diff
@@ -724,21 +725,24 @@ describe("RAMLib event handling", () => {
       const refundedBid2Res = await getBidDetails(client, refundedBid2);
       const refundedBid3Res = await getBidDetails(client, refundedBid3);
 
-      expect(refundedBid1Res.slotIndex).toBe(null);
+      expect(refundedBid1Res.slotIndex).toBe("5");
+      expect(refundedBid1Res.value).toBe(slot5price.toString());
+      expect(refundedBid1Res.isRemoved).toBe(true);
```

Comment:
nit: recommend adding a check for isRemoved == false at some point above

Reply:
🙏 🙏